celer/homotopy.py

```diff
@@ -246,6 +259,7 @@ def celer_path(X, y, pb, eps=1e-3, n_alphas=100, alphas=None,
     # do not skip alphas[0], it is not always alpha_max
     for t in range(n_alphas):
         alpha = alphas[t]
+
```

Comment:
@mathurinm,
**case where ``n_alphas > 1``**
When fitting ``celer_path`` with ``l1_ratio != 1``, will this implies changes on ``w``  for the following iteration

Reply:
a priori no